packages/shared/fastfetchbot_shared/services/scrapers/zhihu/__init__.py

```
@@ -559,12 +559,12 @@ async def _get_zhihu_article(self):
                     self.raw_content = fix_images_and_links(self.raw_content)
                     self.raw_content = unmask_zhihu_links(self.raw_content)
             except Exception as e:
-                raise Exception("zhihu request failed")
+                raise ScraperError("zhihu request failed")
```

Ruff (0.15.7)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)